src/NvGet.Tools.Shared/Arguments/ConsoleArgsContext.cs

```diff
@@ -53,13 +53,14 @@ internal static OptionSet CreateOptionsFor(ConsoleArgsContext context = null)
 				{ "dryrun", "Runs the updater but doesn't write the updates to files.", TrySet(_ => context.Parameters.IsDryRun = true) },
 				{ "result|r=", "The path to the file where the update result should be saved.", TrySet(x => context.ResultFile = x) },
 				{ "versionOverrides=", "The path to a JSON file to force specifc versions to be used; format should be the same as the result file", TryParseAndSet(LoadOverrides, x => context.Parameters.VersionOverrides.AddRange(x)) },
+				{ "projectProperties=", "The path to a JSON file that lists pairs of csproj property names and corresponding package Id, so that updater can update project properties as necessary", TryParseAndSet(LoadProjectProperties, x => context.Parameters.ProjectProperties.AddRange(x)) },
```

Comment:
I'm not sure I understand the use-case so it might be worth some markdown documentation somewhere?

Reply:
And yes, let me push something to the readme.md for the updater to describe this

Reply:
Could the recipe work for values that are not a `PackageId`? It's just that the code implies that, but I wonder whether this could/should be more generic (like `PropertyName` and `PropertyValue`).

Reply:
No because the PackageId is what's used to look up the version that's going to be set as the value. PropertyValue would be too generic and I don't think we should extend this way unless we see a use for it

Reply:
in other words, we're not injecting the values "Uno.UI", we're injecting the updated version number of the Uno.UI package

Reply:
@jeanplevesque  If this makes sense and you're ok with the PR can you go ahead and resolve this thread and merge this PR.

Reply:
I see! I think the `projectProperties` option could be renamed to better reflect the scope (like `unoProjectProperties` or `unoProperties`) just to clarify the scope of that feature. 

Because with that name I though I could set something like `LangVersion` to `12.0`.